src/frontend/src/handler/explain.rs

```diff
@@ -251,7 +261,11 @@ pub async fn do_handle_explain(
             }
             ExplainType::Physical => {
                 // if explain trace is on, the plan has been in the rows
-                if !explain_trace && let Ok(plan) = &plan {
+                if !explain_trace && let Ok(physical_plan) = &plan {
+                    let plan = match &physical_plan {
+                        PhysicalPlanRef::Stream(plan) => plan as &dyn Explain,
+                        PhysicalPlanRef::Batch(plan) => plan as &dyn Explain,
+                    };
```

Comment:
Is it possible to extract this into a method?

Reply:
Seems unnecessary, since this enum is declared and used only within the method body, and is cast to dyn Explain for only once. Same for `.ctx()`